fractions/microprofile/microprofile-metrics/README.adoc

```diff
@@ -1,4 +1,4 @@
-= MicroProfile Metrics
+= Eclipse MicroProfile Metrics
```

Comment:
Again, can we not change the heading, but instead change the 1st sentence below? (While you're at it, could you remove the "1.0" bit? The sentence  should read `This fraction implements the https://github.com/eclipse/microprofile-metrics/[Eclipse MicroProfile Metrics specification].`